tests/nodeos_run_test.py

```diff
@@ -599,6 +599,33 @@
     if actual != expected:
         errorExit("FAILURE - Wrong currency1111 balance (expected=%s, actual=%s)" % (str(expected), str(actual)), raw=True)
 
+    Print("push transfer action to currency1111 contract with sign skipping option enabled")
+    data="{\"from\":\"currency1111\",\"to\":\"defproducera\",\"quantity\":"
+    data +="\"00.0001 CUR\",\"memo\":\"test\"}"
+    opts="-s -d --permission currency1111@active"
+    trans=node.pushMessage(contract, action, data, opts)
+
+    try:
+        assert(not trans[1]["signatures"])
+    except (AssertionError, KeyError) as _:
+        Print("ERROR: Expected signatures array to be empty due to skipping option enabled.")
+        raise
+
+    try:
+        assert("hex_data" in trans[1]["actions"][0])
+    except (AssertionError, KeyError) as _:
```

Comment:
Can you add checks for `data.from, data.to, data.memo`. We would like to move away from `hex_data` only being there when `data` contains expanded form, so would prefer this test to test for expanded `data` directly.

Reply:
Done